subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeSingleCppProjectIntegrationTest.groovy

```diff
@@ -60,8 +66,29 @@ apply plugin: 'cpp-executable'
 
         project.products.children.size() == 1
         project.products.children[0].path == exe("build/exe/main/debug/app").absolutePath
+
+        when:
+        def resultDebug = newXcodebuildExecuter()
+            .withProject(file("${rootProjectName}.xcodeproj"))
```

Comment:
It seems like all of the users of this need to set this property or `withWorkspace` -- is there a good default we can provide for this? They look really similar to me. I assume `withWorkspace` is when the Gradle build is a multi-project build and `withProject` is when the Gradle build is a single project build?

Reply:
Yes and no. You can still invoke the workspace for single project build and use the project for multi-project build. However, it wouldn't be convenient due to auto complete and added layers of IDE files. With regards to default values, does [this commit](https://github.com/gradle/gradle/pull/2923/commits/e65008148dc6d295f5f85c3b7da55c0973a078ef) address your comment?